mne/preprocessing/tests/test_realign.py

```diff
@@ -31,7 +35,10 @@ def test_realign(ratio_other, start_raw, start_other, stop_raw, stop_other):
     n_events = len(orig_events)
     times = np.arange(len(signal)) / sfreq
     stim = np.convolve(signal, np.ones(int(round(0.02 * sfreq))))[: len(times)]
-    signal = np.convolve(signal, np.hanning(int(round(0.2 * sfreq))))[: len(times)]
+    sig_hann = np.convolve(signal, np.hanning(int(round(hann_len * sfreq))))[
+        : len(times)
+    ]
+    sig_box = np.convolve(signal, np.ones(int(round(box_len * sfreq))))[: len(times)]
```

Comment:
what does this help test, that isn't already tested with `stim` (which is also a boxcar-like signal)?

Reply:
boxcars are meant to provide a data-wise "ground-truth" for the annotations. So that if we see the annotations line up with the boxcar signals we are sure we are doing fine

Reply:
ok, so the annotations are made to coincide with the boxcars, and you're testing that they remain co-incident.  So, can we instead eliminate `stim` from the test then?  Or is it not redundant?

Reply:
`stim` is for `find_event`, which subsequently defines `annotations`. I think it's nice to keep all three channels (`stim` to show event information is preserved after realignment; `hann` to simulate a naive EEG response; `boxcar` to coincide with annotations)

Reply:
I added an additional annotation/boxcar alignment test (at boxcar onsets boxcar data should transit from 0 to 1, and from 1 to 0 at offsets)